### DISCUSSION TITLE
Working jointly on MIMIC data

### DISCUSSION THREAD

**Discussion by ghost** [MIT-LCP/mimic-code#1576] [MIMIC-General] [answered]
Hi MIMIC!

I have a pretty basic question, but could not find any existing discussion on it. I'm part of a team using the MIMIC-IV data for benchmarking. We would like to work collaboratively on the data in a single DB, ensuring that everyone with access has gone through the access authorization process on PhysioNet. 

However, the MIMIC use agreement contains the following very general statement: "I will not share access to PhysioNet restricted data with anyone else." On the face of it, that seems to preclude *any* form of sharing, including me storing the data somewhere where multiple people have access *even if access is restricted to people who are all authorized MIMIC users*. That would seem strange, though, since the people with access could get to the data directly anyway.

Could you perhaps clarify the rules regarding joint work on MIMIC data stored in a single location? In particular, what sort of access controls are required if MIMIC data is stored in a location that can (potentially) be accessed by multiple users (like a DB or a shared drive)? Thanks!

**Comment by alistairewj** [comment, marked as answer]
The goal of that clause is to emphasize that you may not distribute the datasets broadly. Obviously what you are talking about is reasonable, but it's hard to draw a sharp line as to where the boundary is between a small-ish shared lab database like what you refer to and one that is too broadly sharing the data, so the DUA is conservative.

Every user who accesses MIMIC must be credentialed. If you follow that principle you will be fine!

**Reply by ghost** [reply]
Great, thanks!
